projects/common/src/index.ts

```diff
@@ -166,6 +166,7 @@ export interface Content extends WithKey {
     showQuotedHeadline: boolean
     mediaType: MediaType
     sportScore?: string
+    print: boolean
```

Comment:
could we call it `isPrint` for more clarity?

Reply:
or maybe `isPrintOnly`

Reply:
maybe `fromPrint`?